src/components/PlayerSubmissionForm.js

```diff
@@ -1,24 +1,85 @@
 import React, { useState } from 'react';
 import PropTypes from 'prop-types';
-
 import './PlayerSubmissionForm.css';
 
-const PlayerSubmissionForm = () => {
+const PlayerSubmissionForm = (props) => {
```

Comment:
Just a note you're getting a warning when the application runs (when you type in a textbox) here because the original value of the forms are `undefined` and then when the user types in them they get set.  

It's because the keys in `props.fields`, don't match the keys in `wordFields`.